SU-SWS/decanter
# Set up color maps for Decanter and refactor main nav colors to refer to color maps

# READY FOR REVIEW

# Summary
- Set up color maps for Decanter - a main one `$core-colors` and one for main-nav `$main-nav-colors`. There is already an existing one called `$alert-colors`.
- Add a function `color` to get colors from color maps.
- Refactor main-nav and a few other components to pull colors from the color map
- Also included issue 422 - changed all reference to $color-dark-red to $color-bright-red for accessibility

# Needed By (Date)
- Whenever

# Urgency
- Not urgent

# Steps to Test

1. Pull this branch
2. See that the main nav, buttons, brand bar, logo, global footer, alerts, links and all variants still have the correct colors.
3. Look at the new color maps in utilities/variables and the function `color`.
4. See that all places in main-nav with $color-dark-red have been changed to $color-bright-red.

# Affected Projects or Products
- Decanter

# Associated Issues and/or People
- https://github.com/SU-SWS/decanter/issues/386
- https://github.com/SU-SWS/decanter/issues/420
- https://github.com/SU-SWS/decanter/issues/422
